docs/releasehistory.rst

```diff
@@ -376,17 +399,17 @@ Bugfixes
   We anticipate that this will reduce the number of times you need to
   type ``allow_undefined_stereo=True`` when processing molecules that do not actually contain stereochemistrty.
 - `Issue #513 <https://github.com/openforcefield/openforcefield/issues/513>`_: The
-   :py:meth:`Molecule.to_rdkit <openforcefield.topology.Molecule.to_rdkit>` now re-sets the aromaticity model
-   after sanitizing the molecule.
+  :py:meth:`Molecule.to_rdkit <openforcefield.topology.Molecule.to_rdkit>` now re-sets the aromaticity model
+  after sanitizing the molecule.
 - `Issue #500 <https://github.com/openforcefield/openforcefield/issues/500>`_: The
-   :py:meth:`Molecule.find_rotatable_bonds <openforcefield.topology.Molecule.find_rotatable_bonds>` has been added
-   which returns a list of rotatable :py:class:`Bond <openforcefield.topology.Bond>` instances for the molecule.
+  :py:meth:`Molecule.find_rotatable_bonds <openforcefield.topology.Molecule.find_rotatable_bonds>` has been added
+  which returns a list of rotatable :py:class:`Bond <openforcefield.topology.Bond>` instances for the molecule.
 - `Issue #491 <https://github.com/openforcefield/openforcefield/issues/491>`_: We can now parse large molecules without hitting a match limit cap.
 - `Issue #474 <https://github.com/openforcefield/openforcefield/issues/474>`_: We can now  convert molecules to InChI and
-   InChIKey and from InChI.
+  InChIKey and from InChI.
 - `Issue #523 <https://github.com/openforcefield/openforcefield/issues/523>`_: The
-   :py:meth:`Molecule.to_file <openforcefield.topology.Molecule.to_file>` can now correctly write to `MOL` files in
-   line with the support file type list.
+  :py:meth:`Molecule.to_file <openforcefield.topology.Molecule.to_file>` can now correctly write to `MOL` files in
```

Comment:
Not sure if Mol was intended to have 0, 1, or 2 ` but PDB below (and elsewhere) has 0. May be inconsistent across documentation elsewhere (it's near impossible to not be)